src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -710,7 +710,7 @@ public static void severe(Logger logger, Throwable e, String message){
         if(e != null) {
             StringWriter sw = new StringWriter();
             e.printStackTrace(new PrintWriter(sw));
-            logger.finer(message + ": " + sw.toString());
+            logger.warning(message + ": " + sw.toString());
```

Comment:
Eventually I will remove this from the PR since it's unrelated, but:

A customer running on 2.8.1 mentioned that they were only seeing "An unexpected error occurred", but no other information — in particular, no stack trace. I wonder if it's because `FINER` level logs may not show up in all situations?

Should we change the log level here so that stack traces always show up?

Interestingly there's no `logger.error()`, so `logger.warning()` is the closest log level, but I believe it could still be disabled in practice. WDYT?

Reply:
Can we try with warning instead of finer and verify that the stack trace does show up?

Reply:
Yes. This is independent of the detailed error messages though so I've dropped it from this PR and will submit something separately.